bedrock/firefox/templates/firefox/switch.html

```diff
@@ -0,0 +1,93 @@
+      <li>{{ _('Enjoy the web faster, all set up for you.') }}</li>
+    </ol>
+    {{ download_firefox(alt_copy=_('Download and switch'), download_location='primary cta') }}
+
+    <footer>
+      <h2>{{ _('Note:') }}</h2>
+      <ul>
+        <li>
+          {{ _('You can use Firefox and still have Chrome. It’s true, Chrome won’t change on your machine one bit.') }}
+        </li>
+
+        <li>
+          <details>
+            <summary>{{ _('Share with your friends how to switch to Firefox') }}</summary>
+
+            {% set share_url = settings.CANONICAL_URL + canonical_path %}
+
+            <ul class="share-options">
+              <li class="share-opt-facebook">
+                <a data-link-type="link" data-link-name="Share on Facebook" href="https://www.facebook.com/sharer/sharer.php?s=100&p[url]={{ share_url|urlencode }}">{{ _('Share to Facebook') }}</a>
+              </li>
+
+              <li class="share-opt-email">
+                <details>
+                  <summary>{{ _('Send an email') }}</summary>
+                  {# L10n: the string below is for a referral email subject line. #}
+                  {% set email_subject = _('Switch to Firefox')|urlencode %}
+
+                  {# L10n: the string below is for a referral email message body. Use \n\n for line breaks in the message as appropriate. #}
+                  {% set email_body = _('Hey, \n\nFirefox makes switching from Chrome really fast. I like it a lot, and you should try it. \n\nCheck it out and let me know what you think: ')|urlencode %}
+                  <ul class="email-links">
+                    <li><a class="mail-gmail" data-link-type="link" data-link-name="Send an email - Gmail" href="{{ 'https://mail.google.com/mail/?view=cm&fs=1&su=%s&body=%s%s'|format(email_subject, email_body, share_url|urlencode)|e }}">{{ _('Gmail') }}</a></li>
+                    <li><a class="mail-yahoo" data-link-type="link" data-link-name="Send an email - Yahoo" href="{{ 'https://compose.mail.yahoo.com/?subject=%s&body=%s%s'|format(email_subject, email_body, share_url|urlencode)|e }}">{{ _('Yahoo Mail') }}</a></li>
+                    <li><a class="mail-outlook" data-link-type="link" data-link-name="Send an email - Outlook" href="{{ 'https://mail.live.com/mail/EditMessageLight.aspx?n=&subject=%s&body=%s%s'|format(email_subject, email_body, share_url|urlencode)|e }}">{{ _('Outlook.com') }}</a></li>
+                    <li><a class="mail-other" data-link-type="link" data-link-name="Send an email - Other" href="{{ 'mailto:?subject=%s&body=%s%s'|format(email_subject, email_body, share_url|urlencode)|e }}">{{ _('Other') }}</a></li>
+                  </ul>
+                </details>
+              </li>
+
+              <li class="share-opt-twitter">
+                {# L10n: the string below is for a referral Twitter message. #}
+                {% set tweet_text = _('🔥 Firefox makes switching from Chrome really fast. Try it out!') %}
```

Comment:
@flodolo do you see any issues using an emoji in a translated string like this?

Reply:
Double checked with Théo, he [used emojis](https://github.com/mozilla-l10n/engagement-l10n/blob/master/en-US/social/2017/iot_survey.lang) in the past and we didn't have issues in our toolchain